build/images/scripts/install_cni

```diff
@@ -27,6 +27,9 @@ install -m 755 /opt/cni/bin/portmap /host/opt/cni/bin/portmap
 # Install bandwidth CNI binary file, It is required to support traffic shaping.
```

Comment:
Not your code, but would you mind change "," to "." in your PR?

Reply:
sure. took care.